### PR TITLE
Enhancement/live update message

### LINKED ISSUE #403: Enhancement/Real Time update message
* **I'm submitting a ...**
    - [x] feature request
    - [ ] bug report

* **Feature Request**
  Include message on tx table, to prompt user to refresh the page if new block came in

### PR DESCRIPTION
**Pull request by olgakup**
This fixes #403

This is ready for review. PLEASE put in the correct traffic header: * or <http://ethvm.${DOMAIN}>

In This PR: 
- Implemented App Live Update Message
- Removed Vue Icon from tab and updated description
- Updated GitHub link in Footer and  BTC donation address

### CODE REVIEW

File: apps/explorer/src/core/components/ui/AppLiveUpdate.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { Vue, Component, Mixins } from 'vue-property-decorator'
+import { Events, BlockMetrics } from 'ethvm-common'
+import { StringConcatMixin } from '@app/core/components/mixins'
+
+@Component
+export default class AppLiveUpdate extends Mixins(StringConcatMixin) {
+  lastBlock: string = null
+  update = false
+  newTxs = 1000000
+  /*
```
Comment (aldoborrero) [inline apps/explorer/src/core/components/ui/AppLiveUpdate.vue]: Add space between variable newTxs and comment

File: apps/explorer/src/core/components/ui/AppLiveUpdate.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      const newBlock = this.$store.getters.blockMetrics.top()
+      if (newBlock) {
+        !this.lastBlock ? (this.lastBlock = newBlock.number) : this.process(newBlock)
+      }
+    })
+  }
+
+  beforeDestroy() {
+    this.$eventHub.$off([Events.NEW_BLOCK_METRIC])
+  }
```
Comment (aldoborrero) [inline apps/explorer/src/core/components/ui/AppLiveUpdate.vue]: Add space between } and comment

File: docker-compose.yaml [hunk 3]
```diff
@@ -91,12 +91,10 @@ services:
       - "traefik.enable=true"
       - "traefik.default.protocol=http"
       - "traefik.frontend.rule=Host:api.ethvm.${DOMAIN}"
-      - "traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:http://ethvm.${DOMAIN}||Access-Control-Allow-Credentials:true"
```
Comment (aldoborrero) [inline docker-compose.yaml]: Why is this line removed?

File: apps/explorer/src/core/components/ui/AppLiveUpdate.vue (hunk 1 above)
Comment (olgakup) [reply apps/explorer/src/core/components/ui/AppLiveUpdate.vue]: Done

File: docker-compose.yaml (hunk 3 above)
Comment (olgakup) [reply docker-compose.yaml]: Hey i was using it instead of Cors, so i can use Chrome. I will put it back when done :)

Comment (olgakup) [reply docker-compose.yaml]: This is what I am using      
 - 'traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:http://localhost:8080||Access-Control-Allow-Credentials:true'

Comment (olgakup) [reply docker-compose.yaml]: Sorry for gazilion messages, in develop its:  - 'traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:*||Access-Control-Allow-Credentials:true'
Can you please put the correct one. Not sure which one to use * or http://ethvm.${DOMAIN}

File: apps/explorer/src/core/components/ui/AppLiveUpdate.vue (hunk 2 above)
Comment (olgakup) [reply apps/explorer/src/core/components/ui/AppLiveUpdate.vue]: done

**Comment by olgakup** [comment]
This is ready for review.

**Comment by aldoborrero** [comment]
@olgakup I think we should take into consideration other factors. For example:

- If, as a user, I'm on PageBlocks.vue and new blocks appears and I click on those I'm successfully refreshing the page and loading newer blocks but I'm not updating the total amount of new blocks that we currently have (so the number of total blocks will be always the same).
- If, as a user, I'm on PageTxs.vue and new txs arrives, happens the same, the total number of Txs is never updated, and also the pagination will be using the old value (the field from).

We need to fix that before merging.

**Comment by olgakup** [comment]
@aldoborrero I added total blocks/txs update. I also fixed live component logic implementation.

Let me know if I am missing anything. Otherwise this should be ready.
